src/components/Header.js

```diff
@@ -22,15 +22,17 @@ class Header extends React.Component {
     super(props);
     const { description, image, twitterHandle, title } = props;
     const meta = {
-      'twitter:site': 'opencollect',
-      'twitter:creator': twitterHandle,
+      'twitter:site': '@opencollect',
+      'twitter:creator': twitterHandle ? `@${twitterHandle}` : ' ',
       'fb:app_id': '266835577107099',
-      'og:image': image,
+      'og:image': image ? image : 'https://opencollective.com/static/images/opencollectiveicon-240x240@2x.png',
       description: truncate(description, 256),
-      'twitter:card': 'summary',
-      'twitter:title': `${title} - Open Collective`,
-      'twitter:description': description,
-      'twitter:image': image,
+      'og:description': truncate(description, 256),
+      'twitter:card': 'summary_large_image',
+      'twitter:title': title ? `${title} - Open Collective` : 'Open Collective',
+      'twitter:description': truncate(description, 256),
+      'twitter:image': image ? image : 'https://opencollective.com/static/images/opencollectiveicon-240x240@2x.png',
```

Comment:
it's usually not a good practice to repeat the same string twice.
Keep it in a variable, e.g. `defaultImage = ...`

Reply:
I understand